packager/react-packager/src/Resolver/index.js

```diff
@@ -239,4 +241,12 @@ function defineModuleCode(moduleName, code) {
   ].join('');
 }
 
+function definePolyfillCode(code) {
+  return [
+    '(function(global) {\n',
```

Comment:
Please remove the line break here, otherwise line numbers in redboxes will be off by one for polyfills.